data/data/openstack/main.tf

```diff
@@ -53,9 +53,11 @@ module "masters" {
     var.openstack_master_extra_sg_ids,
     [module.topology.master_sg_id],
   )
-  root_volume_size = var.openstack_master_root_volume_size
-  root_volume_type = var.openstack_master_root_volume_type
-  server_group_id  = var.openstack_master_server_group_id
+  root_volume_size           = var.openstack_master_root_volume_size
+  root_volume_type           = var.openstack_master_root_volume_type
+  server_group_id            = var.openstack_master_server_group_id
+  additional_network_ids     = var.openstack_additional_network_ids
```

Comment:
Can we make this variable name consistent with the other variables? e.g `openstack_master_extra_network_ids` or something similar?

Reply:
yeah, I forgot that we already do the concatination! I updated the patch as you asked